.travis.yml

```diff
@@ -38,6 +38,9 @@ notifications:
   email:
     on_success: never
     on_failure: change
-  webhooks:
-    urls:
-      secure: LuDOP1T+HXg6E4+ijpSY0YuUH9cwPUp4V9emq2rXJ+oQ2i6zMA5bTufF/d6MqmzVhYrYKaiqII1DyMheTzn7ulZlTjbAsG0ADlbtBzovrMhlRE1mV4e+CmJJHXVMyDI7Q9vkfqILng+PKrERY+OypHbe9DmgQQ2C295aJq0eogxFpGWtGCDl1R5kfJJBfKwdAbmwTjnJFStNjUFTuJ6efZTy8kPam2UbYmiIAAXmMhKtz4rva4cA1DO7gEU64ldp27OZGMAASM/2n9z0BYP1ecQJIxykG3jUYwShZ3vq86E8TuHpdZNhU0w8dpx96q9XnhiY4mbruIjVGA9qWjm0fiacLjzlPfnFT4V3p4P/KdHXXAQoUQNlqNx7ydOKeTqzOMhlydMSix/NxYLM45eLfL+CsU1HJyZSBO88M4j9AHVr8yYuHAzWGpYvuT4JjddTkjxnm1jFAzBIgMkADxr/F/LzluOuhBHWHNxbbFtciOFFg4q7ave6Yob+kGYAf7KmiUzYGwOgGZUgltVgF4NLwvYuEaK3gai7ZIJhuLUGkkNJkZuW3gOJQiMOW9J9v8rOVWgtry/LsGeLyam3JaI2COKGxVMY/iz49jxxnhrVH9wUTVL39Cn24FK33h9T114ZYoxyj6AIeJedCWGkdHrxinkoJDZ002+ewI8aVoYJvvA=
+  slack:
+    on_success: change
+    on_failure: always
```

Comment:
Probably want to add `on_pull_requests: false` too

Reply:
I cannot find detailed documentation on this. Can you elaborate what this would change to the current behaviour? As we are only doing builds through pull requests, I'm not sure I understand what the intention is.

Reply:
> I cannot find detailed documentation on this.

https://docs.travis-ci.com/user/notifications/#Notifications-of-PR-builds

> Can you elaborate what this would change to the current behaviour?

Only send Slack notifications for master branch builds, not pull request builds.

> As we are only doing builds through pull requests, I'm not sure I understand what the intention is.

Incorrect:

![image](https://user-images.githubusercontent.com/36432/31720125-83c728f2-b3ca-11e7-92bf-d60d4328d30a.png)

The intention is to only receive notifications for the master branch (e.g. nightly cron build). When someone is working on a pull request, we don't want to spam the Slack channels.

Reply:
> https://docs.travis-ci.com/user/notifications/#Notifications-of-PR-builds

Yes, I found that documentation link as well, but it does not really contain any details.

> The intention is to only receive notifications for the master branch (e.g. nightly cron build). When someone is working on a pull request, we don't want to spam the Slack channels.

Ah, right, my bad. I'll add this setting.